firefox_puppeteer/ui/toolbars.py

```diff
@@ -0,0 +1,337 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from marionette import Wait, By
+
+from ..api.keys import Keys
+from ..api.l10n import L10n
+from ..base import BaseLib
+from ..decorators import use_class_as_property
+
+
+class NavBar(BaseLib):
+
+    @use_class_as_property('ui.toolbars.LocationBar')
+    def locationbar(self):
```

Comment:
Ok, but now that this is written sorting them by hand seems like a horrendous use of time. Ctr-f works fine!